instrumentation-tests/src/androidTest/java/com/mapbox/navigation/instrumentation_tests/core/EvOfflineTest.kt

```diff
@@ -61,6 +66,7 @@ class EvOfflineTest : BaseCoreNoCleanUpTest() {
             withoutInternet {
                 val routes = navigation.requestRoutes(testRoute.routeOptions)
                 assertTrue(routes is RouteRequestResult.Failure)
+                delay(10_000) // Otherwise mock web server can't start on the same port
```

Comment:
Why did it work before?

Reply:
removed, I believe it should work again without it